pkg/dispatcher/config.go

```diff
@@ -210,6 +211,54 @@ func LoadConfig(configPath string) (*Config, error) {
 	return config, nil
 }
 
+func (config *Config) JobsWithMultiOccurrence() []string {
```

Comment:
if itr's exported, it should have GoDoc. Also, I think that because this takes no input, this should be just called `Validate`, and return an appropriate error, not let the callsite do it.

Reply:
Updated.